server/users/views.py

```diff
@@ -1,42 +1,34 @@
 from django.shortcuts import render
 from django.http import JsonResponse
+from django.http import HttpResponse
 
 from rest_framework.decorators import api_view
 from rest_framework.response import Response
 from rest_framework import status
 from .serializers import UserSerializer
-
-from users.models import User
+from .models import Users
 
 from pymongo import MongoClient
-from src.settings import get_key
+from config.settings import get_key
 import datetime 
+import certifi
  
-conn = MongoClient('mongodb+srv://'+get_key("MongoDB_UserName")+':'+get_key("MongoDB_PWD")+'@twigerdb.oyqtu.mongodb.net/'+get_key("MongoDB_DBName")+'?retryWrites=true&w=majority')
+conn = MongoClient(get_key("Driver_Code"), tlsCAFile=certifi.where())
 db = conn.get_database(get_key("MongoDB_DBName"))
-collection = db.twigerDB.User
-
-print('conn: ', conn)
-
-@api_view(['GET'])
-def apiOverview(request):
-    api_urls = {
-        'SignUp' : '/',
-    }
-
-    return Response(api_urls)
+collection = db.Users
 
 
 @api_view(['POST'])
 def signUp(request):
     serializer = UserSerializer(data = request.data)
-
+    
     if serializer.is_valid():
-        serializer.save()
+        if list(collection.find({"email": serializer.data["email"]})): 
+            return Response({"message": "사용중인 이메일 입니다."}, status=status.HTTP_400_BAD_REQUEST)
+
         collection.insert_one(serializer.data)
-        print("Sign up success")
+        pid = list(collection.find())[::-1][0]['_id']
         
-        return Response(serializer.data, status=status.HTTP_201_CREATED)
+        return Response({"pid": str(pid), "messge": "Sign up success"}, status=status.HTTP_201_CREATED)
```

Comment:
messge 오타 수정 부탁드립니다.
```suggestion
        return Response({"pid": str(pid), "message": "Sign up success"}, status=status.HTTP_201_CREATED)
```